run_dir/design/running_notes_tab.html

```diff
@@ -51,19 +48,22 @@ <h4>Preview</h4>
             <label>Search :</label>
             <input type="text" class="form-control" id="rn_search" size=30 />
             <label>Filter :</label>
-            <button id="rn_cat_all" type="button" class="btn btn-xs btnCatFilter glow">All</button>
-            <button id="rn_cat_workset" type="button" class="btn btn-xs btn-primary btnCatFilter">Workset</button>
-            <button id="rn_cat_flowcell" type="button" class="btn btn-xs btn-success btnCatFilter">Flowcell</button>
-            <button id="rn_cat_meeting" type="button" class="btn btn-xs btn-info btnCatFilter">Meeting</button>
-            <button id="rn_cat_decision" type="button" class="btn btn-xs btn-info btnCatFilter">Decision</button>
-            <button id="rn_cat_bioinfo" type="button" class="btn btn-xs btn-warning btnCatFilter">Bioinformatics</button>
-            <button id="rn_cat_uscom" type="button" class="btn btn-xs btn-danger btnCatFilter">User Communication</button>
-            <button id="rn_cat_uscom" type="button" class="btn btn-xs btn-danger btnCatFilter">Invoicing</button>
-            <button id="rn_cat_imp" type="button" class="btn btn-xs btn-imp btnCatFilter">Important</button>
-            <button id="rn_cat_devi" type="button" class="btn btn-xs btn-devi btnCatFilter">Deviation</button>
+            <select class="form-control" id="rn_category">
+            <option value="All">All</option>
+            <option value="Workset">Workset</option>
+            <option value="Flowcell">Flowcell</option>
+            <option value="Decision">Decision</option>
+            <option value="Bioinformatics">Bioinformatics</option>
+            <option value="User Communication">User Communication</option>
+            <option value="Administration">Administration</option>
+            <option value="Important">Important</option>
+            <option value="Deviation">Deviation</option>
+            <option value="Lab">Lab</option>
+            </select>
```

Comment:
* Whitespace
* If you don't specify a `value` for an `<option>` then it defaults to the text value in the dropdown. Here that is identical to all the values, so you can skip them to make it easier to read / shorter.

```suggestion
            <select class="form-control" id="rn_category">
                <option>All</option>
                <option>Workset</option>
                <option>Flowcell</option>
                <option>Decision</option>
                <option>Bioinformatics</option>
                <option>User Communication</option>
                <option>Administration</option>
                <option>Important</option>
                <option>Deviation</option>
                <option>Lab</option>
            </select>
```